builders/xcm/xtokens.md

```diff
@@ -0,0 +1,244 @@
+
+If you've [checked the prerequisites](#xtokens-check-prerequisites), head to the extrinsic page of [Polkadot JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/extrinsics){target=_blank} and set the following options:
+
+1. Select the account from which you want to send the XCM
+2. Choose the **xTokens** pallet
+3. Choose the **transferMultiasset** extrinsic
+4. To define the XCM asset multilocation, you have to target `UNIT` in the relay chain from Moonbase Alpha as the origin. Each chain sees its own asset differently. Therefore, you will have to set a different asset multilocation for each destination. For this example, the relay chain token can be defined as:
+
+    | Parameter | Value |
+    |:---------:|:-----:|
+    |  Version  |  V1   |
+    |  Parents  |   1   |
+    | Interior  | Here  |
+
+5. Set the asset type as **Fungible**
+6. Set the number of tokens to send. For this example, you are sending 1 `xcUNIT`, but you have to account for the 12 decimals
+7. To define the XCM destination multilocation, you have to target an account in the relay chain from Moonbase Alpha as the origin. Therefore, set the following parameters:
+
+    | Parameter |     Value     |
+    |:---------:|:-------------:|
+    |  Version  |      V1       |
+    |  Parents  |       1       |
+    | Interior  |      X1       |
+    |    X1     |  AccountId32  |
+    |  Network  |      Any      |
+    |    Id     | TargetAccount |
+
+8. Set the destination weight to `1000000000`. Note that on Moonbase Alpha, each XCM instruction costs around `100000000` weight units. A `transferMultiasset` consists of 4 XCM instructions, so a destination weight of `400000000` should be enough
+9. Click the **Submit Transaction** button and sign the transaction
+
+!!! note
+    The encoded call data for the extrinsict configured above is `0x1e010100010000070010a5d4e80101010100c4db7bcb733e117c0b34ac96354b10d47e84a006b9e7e66a229d174e8ff2a06300ca9a3b00000000`.
+
+![XCM X-Tokens Transfer Extrinsic](/images/builders/xcm/xtokens/xtokens-3.png)
+
+Once the transaction is processed, the **TargetAccount** should have received the transferred amount minus a small fee that is deducted to execute the XCM on the destination chain. On Polkadot.js Apps, you can check the relevant extrinsics and events in [Moonbase Alpha](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/explorer/query/0x7711a9bb672782acf6702ebb235cdcbd982d536835e6d00fb07ba716eb1ec982){target=_blank} and the [relay chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/explorer/query/0x03d402f171aa4ab52d6c72d693fb6e76399b88fd44587f748aa685e9b53727ea){target=_blank}.
+
+## X-Tokens Precompile {: #xtokens-precompile}
```

Comment:
Would this section be better as its own page in [the precompiles section](https://docs.moonbeam.network/builders/build/canonical-contracts/precompiles/)  and then just be linked from this page?